repair/repair.cc

```diff
@@ -1289,17 +1289,16 @@ future<> repair_service::sync_data_using_repair(
     });
 }
 
-future<> repair_service::do_sync_data_using_repair(
-        sstring keyspace,
-        dht::token_range_vector ranges,
-        std::unordered_map<dht::token_range, repair_neighbors> neighbors,
-        streaming::stream_reason reason,
-        std::optional<tasks::task_id> ops_uuid) {
-    seastar::sharded<replica::database>& db = get_db();
+future<> tasks::data_sync_repair_task_impl::run() {
```

Comment:
Ditto patch subject / commitlog.

It would be clearer to describe this as e.g. `repair: create task for data sync` or `repair: add data_sync_repair_task_impl`